src/CONST/LOCALES.ts

```diff
@@ -18,40 +24,68 @@ const UPCOMING_LOCALES = {
     PL: 'pl',
     PT_BR: 'pt-BR',
     ZH_HANS: 'zh-hans',
```

Comment:
`NL` and `JA` locales are missing, and we're accessing them below in `LOCALE_TO_LANGUAGE_STRING`.

I think we should include them, no?

Reply:
Oops sorry, GH confused me. Glad everything is okay then!